README.md

```diff
@@ -1,24 +1,75 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Hello rails backend
 
-Things you may want to cover:
+![image](https://user-images.githubusercontent.com/57421823/148772553-3e248246-90fc-4abf-908d-cc7da20a9525.png)
 
-* Ruby version
+Backend to demonstrate the functionality and interactivity with a frontend to consume it. 
 
-* System dependencies
 
-* Configuration
 
-* Database creation
+## Built With
 
-* Database initialization
+- Ruby
+- Rails
+- PostgreSQL
 
-* How to run the test suite
+## Getting Started
 
-* Services (job queues, cache servers, search engines, etc.)
+To get a local copy up and running follow these simple example steps.
 
-* Deployment instructions
+### Prerequisites
 
-* ...
+- Ruby and PosgreSQL must be installed.
+- PostgreSQL server must be running.
+
+### Setup
+
+clone this repo with:
+
+``` ruby
+git clone https://github.com/Meltrust/hello-rails-backend.git
+```
+
+### Install
+
+``` ruby
+run `bundle install`
+run `npm install`
+```
+
+- In your terminal input 
+```ruby
+rails db:migrate
+rails server -p 3001
+```
```

Comment:
- [ ] You have missed two steps, which is` rails db:create`, just before migration and `rails db:seed` after migration. Please mention that so that user can install the backend project locally without any issue.

Reply:
This change has been done well